examples/petstore/services/in_memory_pets_test.go

```diff
@@ -38,7 +41,14 @@ func TestInMemoryPets(t *testing.T) {
 	t.Run("can get all pets", func(t *testing.T) {
 		pets, err := service.GetAllPets()
 		require.NoError(t, err)
-		require.Len(t, pets, 1)
+		require.Len(t, pets, 2)
+	})
+
+	t.Run("can get all pets by age", func(t *testing.T) {
+		pets, err := service.GetAllPetsByAge()
+		require.NoError(t, err)
+		require.Equal(t, "kitkat", pets[1][0].Name)
+		require.Equal(t, "payday", pets[3][0].Name)
```

Comment:
I'm unsure about pets size (4), but this should avoid panic if there is a later refactoring

```suggestion
		require.NoError(t, err)
		require.Len(t, pets, 4)
		require.Equal(t, "kitkat", pets[1][0].Name)
		require.Equal(t, "payday", pets[3][0].Name)
```